assets/static/js/main.js

```diff
@@ -113,5 +125,7 @@ https://github.com/spyder-ide/lektor-icon/blob/master/NOTICE.txt
     mobileMenuOutsideClick();
     mainMenuSticky();
     setFullHeight();
+    fixInnerPage();
+    window.addEventListener("resize", fixInnerPage);
```

Comment:
Everywhere else uses the JQuery `$(window).on("resize", function)` method, and adds the event handler as part of the respective top-level function called here, instead of inside this main block itself. It would be nice to replace JQuery at some point, and its certainly not above consideration to revisit keeping this to just function calls, but for this PR seems best to just be consistent rather than be the one odd one out.

Reply:
Thank you. Solved in [a2c476f](https://github.com/spyder-ide/lektor-icon/pull/26/commits/a2c476f42e86bbc8b74aaa5fa4ad41c92146be3a)